Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/SparkBatchRemoteDebugJob.java

```diff
@@ -485,7 +485,9 @@ static SparkBatchRemoteDebugJob factory(
         driverOption = (carriedDriverOption.trim() + " " + driverDebugOption).trim();
         HashMap<String, Object> jobConfigWithDebug = new HashMap<>(submissionParameter.getJobConfig());
         SparkConfigures sparkConfigWithDebug = new SparkConfigures(sparkConf);
+
         sparkConfigWithDebug.put(sparkJobDriverJvmOptionConfKey, driverOption);
+        sparkConfigWithDebug.put("spark.yarn.maxAppAttempts", "1");
```

Comment:
Here I clone a new instance for submission parameters, So, it won't effect the original customer's setting.